snakemake_script/Snakefile

```diff
@@ -0,0 +1,116 @@
+import os
+
+"""
+Snakemake version 7.19.1 
+Python version 3.11.0
+
+To run this code, write in terminal:
+snakemake --use-conda -c 8 -n 
+"""
+
+# define samples
+SAMPLES = ['AI-69_S60', 'AI-70_S61', 'AI-71_S62', 'AI-72_S63', 'AI-73_S64']
```

Comment:
Oksana showed us some hardcoding here. Don't know how to deal with snakemake here, but my ideology to have the folder of files and make a for loop across the files. BTW don't know how to write it here